jcommune-view/jcommune-web-view/src/main/webapp/WEB-INF/jsp/editProfile.jsp

```diff
@@ -122,7 +122,7 @@
             <img id="avatarPreview" src="data:image/jpeg;base64,${editedUser.avatar}" alt=""/>
           </c:when>
           <c:otherwise>
-            <img src="${pageContext.request.contextPath}/users/${editedUser.userId}/avatar" alt=""/>
+            <img src="${pageContext.request.contextPath}/users/${editedUser.userId}/avatar?version=${editedUser.userVersion}" alt=""/>
```

Comment:
I can't know. This part of code is not mine, it was being here before I changed the line. My changes only affect version of avatar.